src/libdredd/src/mutation_replace_expr.cc

```diff
@@ -67,8 +68,18 @@ std::string MutationReplaceExpr::GetFunctionName(
   // input expression. ~ is omitted for all boolean expressions
   // so the type that is included in the function name will be enough
   // in that case.
-  if (optimise_mutations && IsRedundantOperatorInsertion(expr_, ast_context)) {
-    result += "_uoi_optimised";
+  if (optimise_mutations) {
+    if (IsRedundantOperatorInsertion(expr_, ast_context)) {
```

Comment:
I suggest you move the above comment so that it occurs just before (or just inside) the inner `if`, since it doesn't apply to whole block anymore.

Reply:
Done.